### PR TITLE
Templates supplier

### PR DESCRIPTION
**Pull request by goRoman1**
Working templates on supplier

### CODE REVIEW

**Review by DmytroTHR — CHANGES_REQUESTED** [review]
**Wrong branch!** You should merge to "develop" not "main".
Don't forget to `git pull origin develop` into your branch & resolve conflicts if there would be any. Then make PR on "develop".

**Review by goRoman1 — COMMENTED** [review]
branch to merge has been changed

File: migrations/roma.up.sql [hunk 1]
```diff
@@ -0,0 +1,276 @@
+CREATE TABLE IF NOT EXISTS roles
```
Comment (DmytroTHR) [inline migrations/roma.up.sql]: Do we need this file in develop?

**Review by DmytroTHR — APPROVED** [review]
Check if we need roma.sql file in migration

File: migrations/roma.up.sql (hunk 1 above)
Comment (useralexey2000) [reply migrations/roma.up.sql]: agree

File: routing/supplier.go [hunk 2]
```diff
@@ -15,57 +16,58 @@ import (
...
-	{
-		Uri:     `/addScooter/{id}`,
+		Uri:     `/model/{id}`,
 		Method:  http.MethodPost,
 		Handler: addSuppliersScooter,
 	},
 	{
-		Uri:     `/deleteScooter/{id}`,
-		Method:  http.MethodDelete,
+		Uri:     `/delete/{id}`,
```
Comment (useralexey2000) [inline routing/supplier.go:46]: too ambiguous. previous name was better as for me

**Review by useralexey2000 — APPROVED** [review]

File: repositories/supplier.go [hunk 3]
```diff
@@ -1,3 +1,4 @@
+//go:generate mockgen -source=supplier.go -destination=../repositories/mocks/mock_supplier_repository.go -package=mock
```
Comment (avobl) [inline repositories/supplier.go:1]: I guess we should use only singular names of folders. Like in Golang code-style.
'mock' instead of -> 'mocks'

**Pull request merged**